g/grpc-go/grpc-go_ubi_8.4.sh

```diff
@@ -0,0 +1,93 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : grpc-go 
+# Version       : v1.16.0, v1.41.0
+# Source repo   : https://github.com/grpc/grpc-go 
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "grpc-go" ] ; then
+  rm -rf grpc-go
+fi
+
+# Dependency installation
+dnf update
+dnf install -y go git wget
```

Comment:
When I run these commands (without sudo) I get error  "Error: This command has to be run with superuser privileges (under the root user on most systems)." . Are you not running them as non-root user ?

Reply:
I copied the script in docker and running it. I didn't face any issues.

Reply:
Have checked previously submitted PRs merged without sudo.https://github.com/ppc64le/build-scripts/pull/1033/files

Reply:
Including these changes in latest version of script.